js/utils.js

```diff
@@ -2,6 +2,13 @@ const bigInt = require("snarkjs").bigInt;
 const poseidon = require("circomlib").poseidon;
 const eddsa = require("circomlib").eddsa;
 
+function padZeros(str, length) {
+    while (str.length < length) {
+        str = `0${str}`;
+    }
```

Comment:
Suggestion: instead of while loop: str = "0".repeat(length - str.length) + str

Reply:
indeed it is better to fill with `repeat` method